apple/RNCWebView.m

```diff
@@ -435,6 +435,9 @@ - (void)didMoveToWindow
           NSMutableDictionary *sharedRNCWebViewDictionary= [[RNCWebViewMapManager sharedManager] sharedRNCWebViewDictionary];
           sharedRNCWebViewDictionary[_webViewKey] = self;
         }
+    } else if (_webViewKey != nil) {
```

Comment:
nit: I'm wondering if we should pull this logic outside of the `if (_webView == nil)` block starting on line 426, so we can write this logic just once instead of twice. For example, we could do something like this.

```
if (_webView == nil) {
#if !TARGET_OS_OSX
      _webView = [[WKWebView alloc] initWithFrame:self.bounds configuration: wkWebViewConfig];
#else
      _webView = [[RNCWKWebView alloc] initWithFrame:self.bounds configuration: wkWebViewConfig];
#endif // !TARGET_OS_OSX
}

if (_webViewKey != nil) {
  NSMutableDictionary *sharedRNCWebViewDictionary= [[RNCWebViewMapManager sharedManager] sharedRNCWebViewDictionary];
  sharedRNCWebViewDictionary[_webViewKey] = self;
  if (_webView != nil) {
     sharedWKWebViewDictionary[_webViewKey] = _webView;
 }
}
```

What do you think? This is mostly a code style suggestion, and I'm also cool with the current code changes in the PR